bubblewrap.c

```diff
@@ -2201,8 +2207,11 @@ main (int    argc,
   if (proc_fd != -1)
     close (proc_fd);
 
-  if (opt_sync_fd != -1)
-    close (opt_sync_fd);
+  if (!no_reaper)
```

Comment:
I thought of leaking `opt_sync_fd` into the container process so that sync_fd would still work with `--no-reaper` (unless the container process doesn't close it), so that the two options are not mutually exclusive.

I'd say it should be fine to leak `opt_sync_fd`, or do we change to an explicit error?